core/src/components/card/card.ionic.scss

```diff
@@ -0,0 +1,28 @@
+@use "../../foundations/ionic.vars.scss" as tokens;
+@use "../../themes/ionic.mixins" as mixins;
+
+// Ionic Card
+// --------------------------------------------------
+
+:host {
+  --background: #{tokens.$ionic-color-neutral-0};
+  --border-radius: #{tokens.$ionic-border-radius-rounded-small};
+
+  @include mixins.padding(tokens.$ionic-space-base);
+  @include mixins.border-radius(var(--border-radius));
+
+  display: block;
+
+  border: #{tokens.$ionic-border-size-small} solid #{tokens.$ionic-color-neutral-50};
```

Comment:
I updated this based on the Figma design: https://www.figma.com/file/Fun8j9vlydE65HixgBpA7N/%F0%9F%A5%B7-App-X---UI-Components---WIP?type=design&node-id=20317-62539&mode=design&t=z4J80RlRFDwIBeUO-4